cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1282,6 +1287,13 @@ public Secret generateCertificatesSecret(ClusterCa clusterCa, ClientsCa clientsC
         );
     }
 
+    /**
+     * Get the broker certificate information stored in the broker secret.
+     */
+    public CertAndKey getBrokerCertificates(Secret secret, int brokerId) {
+        return  Ca.asCertAndKey(secret, KafkaResources.kafkaPodName(cluster, brokerId));
```

Comment:
Just a nit ... but if you do some other changes, maybe we can get rid of the doublespace?

```suggestion
        return Ca.asCertAndKey(secret, KafkaResources.kafkaPodName(cluster, brokerId));
```